build.sbt

```diff
@@ -1,10 +1,12 @@
+import scoverage.ScoverageKeys
+import uk.gov.hmrc.DefaultBuildSettings
 
-import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
+ThisBuild / majorVersion := 0
+ThisBuild / scalaVersion := "2.13.12"
 
 val appName = "income-tax-submission"
 
 lazy val coverageSettings: Seq[Setting[_]] = {
```

Comment:
Another syntax warning from the IDE:
`Seq[Setting[_]]` can be `Seq[Setting[?]]`